src/pcapi/core/offerers/repository.py

```diff
@@ -45,66 +50,120 @@ def get_filtered_venues(
-        query = query.filter(Venue.managingOffererId == offerer_id)
+        query = query.filter(offerers_models.Venue.managingOffererId == offerer_id)
 
-    return query.order_by(Venue.name).all()
+    return query.order_by(offerers_models.Venue.name).all()
 
 
 def get_api_key_prefixes(offerer_id: int) -> list[str]:
     return [
         prefix or value[:8]
-        for prefix, value in ApiKey.query.filter_by(offererId=offerer_id)
-        .with_entities(ApiKey.prefix, ApiKey.value)
+        for prefix, value in offerers_models.ApiKey.query.filter_by(offererId=offerer_id)
+        .with_entities(offerers_models.ApiKey.prefix, offerers_models.ApiKey.value)
         .all()
         if prefix or value
     ]
 
 
-def find_offerer_by_siren(siren: str) -> Optional[Offerer]:
-    return Offerer.query.filter_by(siren=siren).one_or_none()
+def find_offerer_by_siren(siren: str) -> Optional[offerers_models.Offerer]:
+    return offerers_models.Offerer.query.filter_by(siren=siren).one_or_none()
+
+
+def find_user_offerer_by_validation_token(token: str) -> Optional[offerers_models.UserOfferer]:
+    return offerers_models.UserOfferer.query.filter_by(validationToken=token).one_or_none()
+
+
+def find_offerer_by_validation_token(token: str) -> Optional[offerers_models.UserOfferer]:
+    return offerers_models.Offerer.query.filter_by(validationToken=token).one_or_none()
+
+
+def find_venue_by_id(venue_id: int) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(id=venue_id).one_or_none()
+
+
+def find_venue_by_siret(siret: str) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(siret=siret).one_or_none()
+
 
+def find_venue_by_managing_offerer_id(offerer_id: int) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(managingOffererId=offerer_id).first()
 
-def find_user_offerer_by_validation_token(token: str) -> Optional[UserOfferer]:
-    return UserOfferer.query.filter_by(validationToken=token).one_or_none()
 
+def find_virtual_venue_by_offerer_id(offerer_id: int) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(managingOffererId=offerer_id, isVirtual=True).first()
 
-def find_offerer_by_validation_token(token: str) -> Optional[UserOfferer]:
-    return Offerer.query.filter_by(validationToken=token).one_or_none()
 
+def has_venue_missing_bank_information(offerer_id: int) -> bool:
+    return db.session.query(
+        offerers_models.Venue.query.join(
+            offerers_models.Offerer, offerers_models.Offerer.id == offerers_models.Venue.managingOffererId
+        )
+        .filter(offerers_models.Offerer.id == offerer_id)
+        .filter(offerers_models.Venue.bankInformation == None)
+        .exists()
+    ).scalar()
 
-def find_venue_by_id(venue_id: int) -> Optional[Venue]:
-    return Venue.query.filter_by(id=venue_id).one_or_none()
 
+def is_digital_venue_having_offers(offerer_id: int) -> bool:
```

Comment:
idem, quelque chose comme:
```suggestion
def do_all_digital_venues_have_offers(offerer_id: int) -> bool:
```

Reply:
Je n'étais pas fier de ce nom...
Toujours avec l'idée de préfixer avec is ou has, je propose: `has_digitial_venue_with_at_least_one_offer()` car un offreur ne peut avoir qu'un lieu numérique.

Reply:
S'il n'y a qu'un seul lieu numérique, on peut opter pour `has_digitial_venue_at_least_one_offer`